preview_generator/preview/builder/document__scribus.py

```diff
@@ -31,14 +33,18 @@ class DocumentPreviewBuilderScribus(DocumentPreviewBuilder):
 
     @classmethod
     def check_dependencies(cls) -> bool:
+        logger = logging.getLogger(LOGGER_NAME)
         try:
-            # BUG - 2018/09/26 - Basile - using '-v' on scribus >= 1.5 gives
-            # the version then crash, using FileNotFoundError to make the diff
-            result = check_call(['scribus', '-v'])
+            # INFO - G.M - 2019-01-17 - stderr is redirected to devnull because
+            # scribus print normal information to stderr instead of stdout.
+            result = check_output(['scribus', '-v'], stderr=STDOUT)
             return True
         except FileNotFoundError:
             raise BuilderDependencyNotFound("this builder requires scribus to be available")
-        except CalledProcessError:
+        except CalledProcessError as exc:
+            # TODO - 2018/09/26 - Basile - using '-v' on scribus >= 1.5 gives
+            # the version then crash, using FileNotFoundError to make the diff
+            logger.warning(exc.output)
```

Comment:
Can you add precision about log like `logger.warning("Scribus like missing (Note: scribus >= 1.5 can produce false error on this check): {}".format(exc.output))`

Reply:
yes, this is the minimum, otherwise the log does not explain context and is almost cryptic

Reply:
Done